blackbox/docs/sql/ddl.txt

```diff
@@ -1049,3 +1049,251 @@ means that if you re-create the table using the resulting ``CREATE TABLE``
 statement the settings of the 'old' table may differ from the settings of
 the 'new' table. This is because the table settings are set explicitly on
 creation time.
+
+User defined Functions
+======================
+
+CREATE OR REPLACE
+-----------------
+
+CrateDB supports user-defined functions. See :ref:`ref-create-function`
+for a full syntax description.
+
+These functions can be created, like so::
+
+     cr> CREATE FUNCTION subtract(integer, integer)
+     ...  RETURNS integer
+     ...  LANGUAGE JAVASCRIPT
+     ...  AS 'function(a, b){
+     ...        return a - b;
+     ...      }';
+
+``OR REPLACE`` can be used to replace an existing function::
+
+     cr> CREATE OR REPLACE FUNCTION subtract(integer, integer)
+     ...  RETURNS integer
+     ...  LANGUAGE JAVASCRIPT
+     ...  AS 'function(a, b){
+     ...        return a - b;
+     ...      }';
+
+.. NOTE::
+
+   It is considered bad practice to create functions that have the same name
+   as the CrateDB built-in functions. The CrateDB built-in function will be
+   executed first. However, such user-defined functions can still be called
```

Comment:
will be looked up first when no schema name is provided and therefore hides the user defined function.